bears/rest/RSTcheckBear.py

```diff
@@ -0,0 +1,49 @@
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.bears.requirements.PipRequirement import PipRequirement
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+
+
+@linter(executable="rstcheck",
+        use_stdout=False,
+        use_stderr=True,
+        output_format="regex",
+        output_regex=r'.+\s?:(?P<line>\d+):\s'
+                     r'(?P<message>\((?P<severity>\w+).+)',
+        severity_map={"INFO": RESULT_SEVERITY.INFO,
+                      "WARNING": RESULT_SEVERITY.NORMAL,
+                      "ERROR": RESULT_SEVERITY.MAJOR,
+                      "SEVERE": RESULT_SEVERITY.MAJOR})
+class RSTcheckBear:
+
+    """
+    A Bear for rstcheck.
+    Check <https://pypi.python.org/pypi/rstcheck> for more information.
```

Comment:
Seems cleaner!!

`Check syntax` or `Checks syntax` ? 
Also should I change the variable name for ignore list? It seems too long to me.
Any suggestions on what it could be?